ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -625,15 +652,22 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     //////////////////////////////////
     
     static NSString *identifier = nil;
-
+    
     assert (self.questionStep.isFormatFitsChoiceCells);
     
     identifier = [NSStringFromClass([self class]) stringByAppendingFormat:@"%@", @(indexPath.row)];
     
     ORKChoiceViewCell *cell = [tableView dequeueReusableCellWithIdentifier:identifier];
     
     if (cell == nil) {
-        cell = [_choiceCellGroup cellAtIndexPath:indexPath withReuseIdentifier:identifier];
+        if (_choiceCellGroup) {
```

Comment:
Not a huge fan of all of these cell group ifs, it would be nice if the was a cell group protocol or med choice extended text choice somehow so this could all be handled without the logic branching.